Lib/test/test_pathlib/test_join_windows.py

```diff
@@ -70,17 +75,17 @@ def test_div(self):
         self.assertEqual(p / './dd:s', P(r'C:/a/b\./dd:s'))
         self.assertEqual(p / 'E:d:s', P('E:d:s'))
 
-    def test_str(self):
```

Comment:
Keep/add `test_str` for the existing path types?
Changing `__str__` would probably count as a breaking change.

Reply:
The pathlib ABCs are still private in CPython, and this doesn't affect `pathlib.PurePath` and friends, so I think it's OK. It _will_ require a version bump in my pathlib-abc PyPI package though.